### PR TITLE
Include URLSessionTask in errors.md

### PR DESCRIPTION
**Pull request by schmjop**
Update of errors.md based on [issue 1571](https://github.com/home-assistant/iOS/issues/1571) in the Home Assistant iOS repo.

### CODE REVIEW

File: docs/troubleshooting/errors.md [hunk 1]
```diff
@@ -28,8 +28,12 @@ This error is usually caused by one of the two following issues:
 *   You are missing `default_config:` from your `configuration.yaml` file. It is possible to only add `zeroconf:` to `configuration.yaml` but adding `default_config:` will add [several useful integrations](https://www.home-assistant.io/components/default_config/) along with `zeroconf:`.
 
 ### Response status code was unacceptable: 400
-
 This occurs when the data sent during set up does not meet Home Assistant's expectations. This most commonly occurs in two circumstances:
 
 * When you are running a version of Home Assistant older than the minimum requirement (currently 0.104.0)
 * You have unexpected characters in your device's name. While setting up the Mobile App integration, we attempt to remove non-standard characters and emoji (as of Home Assistant 0.112). However, it is worth simplifying your device name to remove such characters if you are getting this error.
+
+### "URLSessionTask failed with error"
```
Comment (TomBrien) [inline docs/troubleshooting/errors.md:36]:
```suggestion
### URLSessionTask failed with error
```

File: docs/troubleshooting/errors.md [hunk 2]
```diff
@@ -28,8 +28,12 @@ This error is usually caused by one of the two following issues:
...
+This error is usually caused by one of the two following issues:
+*   You have denied local network access to the app. To solve the issue on iOS, open the Home Assistant entry in the system settings and verify that local network is enabled.
+*   You have configured an incorrect `external url` in your `configuration.yaml` file. E.g. when forwarding external port 443 to internal port 8123 in your router settings, you don't have to add a port for the external url, as port 443 is the port used for HTTPS.
```
Comment (TomBrien) [inline docs/troubleshooting/errors.md:39]:
```suggestion
*   You have configured an incorrect external url in your instance. E.g. when forwarding external port 443 to your instance's port (normally 8123), you don't have to append a port to your URL.
```

You can configure the URL setting through either YAML or the UI.

**Review by TomBrien — CHANGES_REQUESTED** [review]
Just a few tweaks to wordings

File: docs/troubleshooting/errors.md (hunk 2 above)
Comment (schmjop) [reply docs/troubleshooting/errors.md:39]: Thanks!

**Comment by TomBrien** [comment]
@schmjop if you're happy with the suggestions, can you please click "Apply Suggestion" under each (you may need to unresolve the comments first). Otherwise let me know and we can discuss what might be best.

**Comment by schmjop** [comment]
My bad, I have merged your requested changes. Thanks for having a look!

**Comment by TomBrien** [comment]
Great, cheers

**Pull request merged**